sound/soc/intel/boards/Kconfig

```diff
@@ -413,7 +413,7 @@ config SND_SOC_INTEL_KBL_RT5660_MACH
 
 endif ## SND_SOC_INTEL_KBL
 
-if SND_SOC_SOF_GEMINILAKE  && SND_SOC_SOF_HDA_LINK
+if SND_SOC_SOF_GEMINILAKE  && SND_SOC_SOF_HDA_AUDIO_CODEC
```

Comment:
probably best if you move the dependency below, so that all users of HDMI rely on the same pattern of 
	depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC

Reply:
@plbossart OK. I will remove "SND_SOC_SOF_HDA_AUDIO_CODEC" in the if statement and use "depends on SND_HDA_CODEC_HDMI && SND_SOC_SOF_HDA_AUDIO_CODEC"